cmd/goreleaser/internal/configure.go

```diff
@@ -45,6 +46,7 @@ var (
 	winContainerArchs = []string{"amd64"}
 	darwinArchs       = []string{"amd64", "arm64"}
 	k8sArchs          = []string{"amd64", "arm64", "ppc64le", "s390x"}
+	ebpfProfilerArchs = []string{"amd64"}
```

Comment:
Does it make sense to add `arm64` here?

```suggestion
	ebpfProfilerArchs = []string{"amd64", "arm64"}
```